crescent/__init__.py

```diff
@@ -23,6 +23,8 @@
     "MinValue",
     "command",
     "option",
+    "hook",
+    "HookResult",
```

Comment:
You're missing some things from typedefs.py. If you look at the example hook file, The `crescent.HookOptionsT` is Any